ang/civicase/case/details/people-tab/directives/case-details-people-tab.directive.html

```diff
@@ -280,21 +311,43 @@ <h3>{{ ts('%1 Contacts', {1: relations.length}) }}</h3>
         </tr>
         <tr ng-repeat="i in [1, 2, 3]" ng-if="isRelationshipLoading">
           <td class="civicase__people-tab__table-column civicase__people-tab__table-column--first">
-            <div class="civicase__loading-placeholder__oneline" style="width: 1em; font-size: 24px; margin-right: 5px;"></div>
+            <div class="civicase__loading-placeholder__oneline"
```

Comment:
Lets move the placeholders to a separate html file and use `ng-include.`

Reply:
A single placeholder file was needed for both cases. The placeholder for the relationships tab was wrong and using the one for the case roles tab fixed it:

![Screen Shot 2019-12-03 at 4 34 40 PM](https://user-images.githubusercontent.com/1642119/70087654-17dbcc80-15eb-11ea-9a9a-38307150c139.png)